tools/fmt/main.rs

```diff
@@ -134,63 +136,47 @@ fn process_markdown_file(source: String, mut file: impl Write) -> std::io::Resul
     return file.write_all(source_slice.as_bytes());
 }
 
-fn process_file(
+fn process_60_file(
     source: String,
     path: std::path::PathBuf,
     mut file: impl Write,
 ) -> std::io::Result<()> {
-    match path.extension() {
-        Some(ext) if ext == "rs" => return process_rust_file(source, file),
-        Some(ext) if ext == "md" => return process_markdown_file(source, file),
-        _ => {}
-    }
-
     let mut diag = BuildDiagnostics::default();
     let syntax_node = i_slint_compiler::parser::parse(source.clone(), Some(&path), &mut diag);
     let len = syntax_node.node.text_range().end().into();
-    visit_node(syntax_node, &mut file, &mut State::default())?;
+    visit_node(syntax_node, &mut file)?;
     if diag.has_error() {
         file.write_all(&source.as_bytes()[len..])?;
         diag.print();
     }
     Ok(())
 }
 
-type State = ();
+fn process_file(
+    source: String,
+    path: std::path::PathBuf,
+    mut file: impl Write,
+) -> std::io::Result<()> {
+    match path.extension() {
+        Some(ext) if ext == "rs" => return process_rust_file(source, file),
+        Some(ext) if ext == "md" => return process_markdown_file(source, file),
+        Some(ext) if ext == "60" => return process_60_file(source, path, file),
```

Comment:
so actually are the files now also called .slint ? i'll look into renaming these in the next days